Assets/MixedRealityToolkit/_Core/Interfaces/IMixedRealityManager.cs

```diff
@@ -24,6 +34,16 @@ public interface IMixedRealityManager
         /// </summary>
         void Update();
 
+        /// <summary>
+        /// Optional Disable function to pause the manager.
+        /// </summary>
+        void Disable();
+
+        /// <summary>
+        /// Optional Enable function to enable / re-enable the manager.
+        /// </summary>
+        void Enable();
```

Comment:
Let's sort these and make them in order:

- Initialize
- Enable
- Update
- Disable
- Destroy

Reply:
Sure